src/components/application_manager/rpc_plugins/sdl_rpc_plugin/test/commands/hmi/get_system_info_response_test.cc

```diff
@@ -134,6 +135,37 @@ TEST_F(GetSystemInfoResponseTest, GetSystemInfo_UpdateCapabilities_Called) {
   command->Run();
 }
 
+TEST_F(GetSystemInfoResponseTest, GetSystemInfo_SaveHardwareVersionToHMICapabilitiesIfPresent) {
+  MessageSharedPtr command_msg = CreateCommandMsg();
+  (*command_msg)[strings::params][hmi_response::code] =
+      hmi_apis::Common_Result::SUCCESS;
+  (*command_msg)[strings::msg_params][hmi_response::capabilities] =
+      (capabilities_);
+  (*command_msg)[strings::msg_params][strings::system_hardware_version] = kHardwareVersion;
+
+  ResponseFromHMIPtr command(CreateCommand<GetSystemInfoResponse>(command_msg));
+
+  EXPECT_CALL(mock_hmi_capabilities_, set_hardware_version(kHardwareVersion));
+
+  ASSERT_TRUE(command->Init());
+  command->Run();
+}
+
+TEST_F(GetSystemInfoResponseTest, GetSystemInfo_DontSaveHardwareVersionToHMICapabilitiesIfAbsent) {
+  MessageSharedPtr command_msg = CreateCommandMsg();
+  (*command_msg)[strings::params][hmi_response::code] =
+      hmi_apis::Common_Result::SUCCESS;
+  (*command_msg)[strings::msg_params][hmi_response::capabilities] =
+      (capabilities_);
+
+  ResponseFromHMIPtr command(CreateCommand<GetSystemInfoResponse>(command_msg));
+
+  EXPECT_CALL(mock_hmi_capabilities_, set_hardware_version(kHardwareVersion)).Times(0);
```

Comment:
@ychernysheva you can replace that constant with the placeholder because in that case you are not expecting this call with any possible argument
```suggestion
  EXPECT_CALL(mock_hmi_capabilities_, set_hardware_version(_)).Times(0);
```

Reply:
We need to save `EXPECT_CALL(mock_hmi_capabilities_, set_hardware_version(kHardwareVersion)).Times(0)` because of other `set_hardware_version` call, which is added in b24cb18